### DISCUSSION TITLE
Marketplace GitHub with area: technologies

### DISCUSSION THREAD

**Discussion by ghost** [community/community#18822] [Other Feature Feedback, Questions, & Ideas]
Hi all!

#### 1. feature-name
Marketplace GitHub with area: technologies

#### 2. feature-description
I would like the github marketplace to have an area where it shows the software/app technologies

#### 2.1. img, before
![image](https://user-images.githubusercontent.com/95204402/174414516-b105699c-eeea-43bd-abcf-1fa24330c3a4.png)

#### 2.2. img, after
![image](https://user-images.githubusercontent.com/95204402/174414390-4b7a2b6e-04f6-4cd9-b68b-217ce50300bb.png)

#### 3. Reference
- https://github.com/marketplace?type=

**Comment by Or1onn** [comment]
Pls, can tou make my topic as answer?
